tests/CMakeLists.txt

```diff
@@ -33,11 +33,13 @@ function(ly_add_utest)
 
     add_test(NAME ${TEST_NAME} COMMAND ${TEST_NAME})
     set_property(TEST ${TEST_NAME} APPEND PROPERTY ENVIRONMENT "MALLOC_CHECK_=3")
+    set_property(TEST ${TEST_NAME} APPEND PROPERTY ENVIRONMENT "TZ=CET+02:00")
```

Comment:
I'm not sure that this is an improvement. I guess that the code leans on this and having the setup of the environment in the code seems much safer to me. Don't get it as a strong objection, just as a note to consider.

Reply:
If you can come up with a better solution, I would only be glad, this also has the disadvantage of not being able to run the test binaries directly as they expect this timezone. The reason why I changed it this way is the advantage of all the tests being unified in their expected timezone, even any new ones written, which I thought is worth all the disadvantages. I was also glad to get rid of the specific test setup and teardown.

Reply:
Then what about moving the sutup of the environment variable from the local setup into the common setup in `utests.h`?

Reply:
Makes sense, moved.